docs/source/transformers/NormalizeNewLines.rst

```diff
@@ -239,3 +239,71 @@ is set to True.
 
             Mid Test
                 My Step 1    args    args 2    args 3
+
+Language header
+----------------
+Files that begins with language marker are handled differently.
+If the section contains only language marker and no more than ``section_lines``, it will be not transformed.
+Following examples will not be formatted since number of empty lines is lower or equal to default ``section_lines``:
```

Comment:
```suggestion
The following examples will not be formatted since the number of empty lines is lower or equal to default value of ``section_lines``:
```